api/web_conn.go

```diff
@@ -126,6 +138,11 @@ func (webCon *WebConn) InvalidateCache() {
 }
 
 func (webCon *WebConn) ShouldSendEvent(msg *model.WebSocketEvent) bool {
+	// IMPORTANT: Do not send event if WebConn does not have a session
+	if webCon.SessionToken == "" {
```

Comment:
Can we change this to webCon.isAuthenticated() so if our auth methods change stuff has a less chance of breaking?